backend/src/test-integration/java/com/crm/api/v1/CarControllerIntegrationTest.java

```diff
@@ -18,27 +18,27 @@
 @SpringBootTest(classes = App.class)
 @AutoConfigureMockMvc
 @WithMockUser
-public class CarControllerIntegrationTest {
+class CarControllerIntegrationTest {
 
     @Autowired
     private MockMvc mvc;
 
     @Test
-    public void shouldReturnCorrectResponseStatusWhenCallingCarsEndpoint() throws Exception {
+    void shouldReturnCorrectResponseStatusWhenCallingCarsEndpoint() throws Exception {
         mvc.perform(get("/cars"))
                 .andExpect(status().is(OK.value()));
     }
 
     @Test
-    public void shouldResponseEntityHasPageNumberEqualTo2WhenParamSizeEqualTo1() throws Exception {
+    void shouldResponseEntityHasPageNumberEqualTo2WhenParamSizeEqualTo1() throws Exception {
```

Comment:
why in the method name is written 'EqualTo2'?

Reply:
IDK it's not my code.